src/components/package-port/CodeEditorHeader.tsx

```diff
@@ -62,28 +89,48 @@ export const CodeEditorHeader: React.FC<CodeEditorHeaderProps> = ({
             description: "Failed to format JSON: invalid syntax.",
             variant: "destructive",
           })
-          return
         }
         return
       }
 
+      const parserMap: Record<string, string> = {
+        js: "babel",
+        jsx: "babel",
+        ts: "typescript",
+        tsx: "typescript",
+        html: "html",
```

Comment:
we don't need html support, and it's another dep on the page. Nobody should be pushing html files to tscircuit.com i would think